go/vt/vtctl/topo.go

```diff
@@ -47,12 +48,18 @@ func init() {
 		commandTopoCat,
 		"[-cell <cell>] [-decode_proto] [-decode_proto_json] [-long] <path> [<path>...]",
 		"Retrieves the file(s) at <path> from the topo service, and displays it. It can resolve wildcards, and decode the proto-encoded data."})
-
 	addCommand(topoGroupName, command{
 		"TopoCp",
 		commandTopoCp,
 		"[-cell <cell>] [-to_topo] <src> <dst>",
 		"Copies a file from topo to local file structure, or the other way around"})
```

Comment:
It sounds like TopoCp would serve the same purpose as TopoPost? Is the intention here to allow for a more surgical change?